submissions/Complexity Corner 4/Complexity Corner 4.md

```diff
@@ -0,0 +1,27 @@
+### Complexity Corner: Beacons <author>The Kool</author>
+
+Welcome back to Complexity Corner!  This week, I'll be taking a slight left on the topic of modules that I've held for the first three articles, and will be going over the beacon, a building completely reliant on modules for its existence.  As before, I will ask three questions about this potentially unfamiliar thing: “What is it?”, “What results can it give?”, and “Where should it be used?”  My goal is to show those of you who haven't used it yet just how easy it is to work with.
+
+What are beacons?
+
+The [Beacon](https://wiki.factorio.com/Beacon) is a building that takes up a space of 3x3 tiles.  It first appears relatively late in your usual playthrough, requiring purple science to research, but the most advanced ingredient is red circuits.  A beacon has two module slots, and will transmit half the effect of those modules across a 9x9 area, reaching anything within 3 tiles of the beacon.  All this is to say, it takes a module and spreads its effect across nearby buildings.
```

Comment:
The first sentence should read something like: beacons are machines that effects other machines in close range. Then you can start explaining more technical things like the size, etc.